verification/client/abi.go

```diff
@@ -37,17 +37,67 @@ type Support struct {
 	RetainParentContext bool
 }
 
-// All DPE profile command codes
-const (
-	CommandGetProfile          CommandCode = 0x1
-	CommandInitializeContext   CommandCode = 0x7
-	CommandDeriveContext       CommandCode = 0x8
-	CommandCertifyKey          CommandCode = 0x9
-	CommandSign                CommandCode = 0xa
-	CommandRotateContextHandle CommandCode = 0xe
-	CommandDestroyContext      CommandCode = 0xf
-	CommandGetCertificateChain CommandCode = 0x10
-)
+// profileCommandCodes holds command codes for a specific revision of the
+// DPE iRoT profile.
+type profileCommandCodes struct {
+	GetProfile          CommandCode
+	InitializeContext   CommandCode
+	DeriveContext       CommandCode
+	CertifyKey          CommandCode
+	Sign                CommandCode
+	RotateContextHandle CommandCode
+	DestroyContext      CommandCode
+	GetCertificateChain CommandCode
+	ExtendTCI           CommandCode
+}
+
+// profileInfo holds constants defined in a specific version of the DPE iRoT
+// profile.
+type profileInfo struct {
```

Comment:
Should we also include max_tci_nodes in here? It could be helpful for future PRs where we support a non-default number of max_tci_nodes.

Reply:
Max tci nodes is a field returned by GetProfile, so I don't think we need to hard-code that. We can read it from the profile response.